resources/hedera-explorer-values.yaml

```diff
@@ -0,0 +1,49 @@
+ingress:
```

Comment:
ingress might require `haproxyIngressController`, let me know if that is the case or if I'm mistaken.

Reply:
@JeffreyDallas

Reply:
I think this ingress is different one vs haproxy-ingress

Reply:
haproxyIngressController is what we used to configure the ingress in hedera mirror node explorer, those ingress value configurations are used here:

https://github.com/hashgraph/hedera-mirror-node-explorer/blob/main/chart/templates/ingress.yaml

As long as it tests all okay we should be good.  self-signed can be tested locally.  I had previously used GKE to test the letsencrypt signed certificates.

Reply:
Test passed with new explorer chart 0.2.1 
with both `selfSignedClusterIssuer` enabled or disabled